config/initializers/assets.rb

```diff
@@ -1,7 +1,7 @@
 # Be sure to restart your server when you modify this file.
 
 # Version of your assets, change this if you want to expire all your assets.
-Rails.application.config.assets.version = '1.00'
+Rails.application.config.assets.version = '1.01'
```

Comment:
Actioncable refused to work on staging. Actually, to be included in application.js. I had to update assets version to fix this.